modules/gapi/src/backends/ie/giebackend.hpp

```diff
@@ -48,7 +28,11 @@ namespace gimpl {
 namespace ie {
 
 struct IECompiled {
+#if INF_ENGINE_RELEASE < 2019020000  // < 2019.R2
     InferenceEngine::InferencePlugin   this_plugin;
+#else
+    InferenceEngine::Core              this_core;
```

Comment:
So do we copy the `IE::Core` per every `IECompiled` or not? Is it implicitly shared? cc: @ilya-lavrenov

Reply:
So I checked the code and IE::Core is implicitly shared. I believe we can count on this now.